service/history/handler.go

```diff
@@ -163,19 +176,40 @@ func (h *Handler) Start() error {
 		}
 	}
 
+	fetcherConfig := h.GetClusterMetadata().GetReplicationConsumerConfig().FetcherConfig
```

Comment:
plz move this initialization logic into a dedicated file, like TimerQueueProcessor.go or TransferQueueProcessor.go

Reply:
done